syntax = "proto3";
option java_package = "io.datanerd.generated.common";
option java_multiple_files = true;
option go_package = "datanerd";

message Product {
    int64 product_id = 1;
    string product_name = 2;
    double product_price = 3;
    ProductStatus product_status = 4;
    string category = 5;
}

enum ProductStatus {
    InStock = 0;
    OutStock = 1;
}